gamemodes/terrortown/gamemode/shared/hud_elements/tttinfopanel/pure_skin_playerinfo.lua

```diff
@@ -38,6 +38,7 @@ if CLIENT then
 		self.lpw = lpw
 		self.sri_text_width_padding = sri_text_width_padding
 		--self.secondaryRoleInformationFunc = nil
+		self.healthpulsate = huds.GetStored(HUDManager.GetHUD()).healthPulsate
```

Comment:
```suggestion
		self.healthpulsate = huds.GetStored("pure_skin").healthPulsate
```

So this is actually something we are missing, because usually you would just add this variable to the saving keys of the element instead of the HUD.

But we currently do not have an extra UI being generated in the F1 settings for those, which we should!
Dunno if you want to walk the extra mile or maybe someone else volunteers to do this in this PR :D 

Because if you use the current hud this might actually return a different one than pure_skin, this is due to HUDs only being a collection of hudelements. For example incomplete HUDs may use this element as a fallback.

Reply:
Okay.. well.. I do understand the thing about HUDManager.GetHUD returning a different hud it it's a fallback.
But what menu do you have in mind? I actually don't 100% get what you mean.